configuration_handling/app.py

```diff
@@ -0,0 +1,15 @@
+from app import app
+from dotenv import load_dotenv
+load_dotenv = load_dotenv
+import os
+
+env=os.getenv("ENV")
+print(f'you are working on {env} environment')
+@app.route('/')
+def hello_world():
+    print(app.config['DB_NAME'])
+    
+    return 'Hello, World!'
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
Whitespace!

Reply:
done.  white space not with space

Reply:
I have also added config.py to git ignore.  is it suppose to be that way?

Reply:
> I have also added config.py to git ignore. is it suppose to be that way?

You should not. These are the setup settings for your app